src/constants.ts

```diff
@@ -24,7 +24,7 @@ export const addresses: IAddresses = {
   4: {
     SQUID_ADDRESS: "0x75d9f9179F95b41236329f85EA1C12DA824A77B1",
     SSQUID_ADDRESS: "0xe8fb1c2Bd5DCC22d4Cd418a85F5B9cc339e8AD47",
-    WSSQUID_ADDRESS: "",
+    WSSQUID_ADDRESS: "0x3b1388eB39c72D2145f092C01067C02Bb627d4BE",
```

Comment:
This is for Rinkeby setup, and since we haven't deployed contract on Rinkeby, this should be left empty.